eureka2-core/src/test/java/com/netflix/eureka2/interests/ChangeNotificationsTest.java

```diff
@@ -21,15 +28,42 @@
  */
 public class ChangeNotificationsTest {
 
+    private static final TestScheduler testScheduler = Schedulers.test();
+
     private static final Interest<String> INTEREST = new FullRegistryInterest<>();
 
     private static final ChangeNotification<String> ADD_A = new ChangeNotification<>(Kind.Add, "A");
     private static final ChangeNotification<String> DELETE_A = new ChangeNotification<>(Kind.Delete, "A");
+    private static final ChangeNotification<String> MODIFY_A = new ChangeNotification<>(Kind.Modify, "A");
     private static final ChangeNotification<String> ADD_B = new ChangeNotification<>(Kind.Add, "B");
     private static final ChangeNotification<String> ADD_C = new ChangeNotification<>(Kind.Add, "C");
+    private static final ChangeNotification<String> MODIFY_D = new ChangeNotification<>(Kind.Modify, "D");
+    private static final ChangeNotification<String> DELETE_E = new ChangeNotification<>(Kind.Delete, "E");
+
+    private static final List<ChangeNotification<String>> FIRST_BATCH = asList(ADD_A, ADD_B);
+    private static final List<ChangeNotification<String>> SECOND_BATCH = asList(DELETE_A, ADD_C);
+
+    private static final List<ChangeNotification<String>> NOTIFICATION_LIST_1 = asList(
+            ADD_A, DELETE_A, ADD_A, ADD_B, MODIFY_A, MODIFY_D, DELETE_E
```

Comment:
For item A the modify here is not really tested as there is no real data modification, so collapse for meaningful modifies are not actually tested. Can we explicitly test for that?